README.md

```diff
@@ -6,8 +6,7 @@ install Python 3.4
 install pip
 
 pip install virtualenv
-virtualenv -p /usr/bin/python3.4 virtualenv
-(måske skal stien til python rettes)
+virtualenv -p $(which python) virtualenv
```

Comment:
The general comment that one uses the version in ones distro for global usage, and have a local usage of a certain program is also relevant: a bit on the same lines as avoiding npm global